app/src/screens/Home/index.tsx

```diff
@@ -51,17 +53,30 @@ const styles = StyleSheet.create({
 const Home: React.FC = () => {
   const dispatch = useDispatch<Dispatch>();
   const user = useSelector((state: State) => state.user);
+  const navigation = useNavigation<StackNavigationProp<'App'>>();
 
   const handleLogout = async () => {
+    navigation.reset({ routes: [{ name: 'Busy' }] });
```

Comment:
Is it really faster to use navigation to display `Busy`, rather than just returning `Busy` component?

Reply:
It's not that it's faster to use navigation, it's that there were issues with constantly swapping screens in and out of the Navigator components. So I solved some issues with race conditions by just having the components always in the Navigator and then navigating between them.